tsconfig.base.json

```diff
@@ -14,9 +14,17 @@
     "skipLibCheck": true,
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
+
     "paths": {
       "cli": ["apps/cli/src/index.ts"]
-    }
+    },
+
+    "inlineSources": true,
+
+    // Set `sourceRoot` to  "/" to strip the build path prefix
```

Comment:
Remove these comments

Reply:
noted. sorry for this change. its might have automatically done by sentry

Reply:
No issues!